ui/StatusQ/tests/CMakeLists.txt

```diff
@@ -2,44 +2,48 @@ cmake_minimum_required(VERSION 3.5)
 
 project(TestStatusQ LANGUAGES CXX)
 
-enable_testing()
-
 set(CMAKE_AUTOMOC ON)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS QuickTest Qml Quick WebChannel WebEngine REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS QuickTest Qml Quick WebChannel WebEngine REQUIRED)
+find_package(Qt5 COMPONENTS QuickTest Qml Quick WebEngine REQUIRED)
 
-set(CMAKE_INCLUDE_CURRENT_DIR ON)
+##############
+# Test Library
+##############
 
-# no need to copy around qml test files for shadow builds - just set the respective define
-add_definitions(-DQUICK_TEST_SOURCE_DIR="${CMAKE_CURRENT_SOURCE_DIR}")
+set(TEST_LIB "${PROJECT_NAME}Lib")
```

Comment:
We agreed some time ago not to parametrize project names anymore and we didn't refactor this yet. Maybe it is a good idea to do it now and use the literal name.

Reply:
I like the idea. The simpler the better :)